src/app/pages/wallet-details/wallet-details.ts

```diff
@@ -82,7 +83,7 @@ export class WalletDetailsPage {
   public navPramss: any;
   public finishParam: any;
   public isScroll = false;
-
+  isFinish = false;
```

Comment:
Should be isFinished? 
Also finish what? Could we use more meaningful name or add comment?

Reply:
remove unused var